src/app.ts

```diff
@@ -3,38 +3,69 @@ import 'module-alias/register';
 import 'express-async-errors';
 import 'dotenv/config';
 
-import express, { Application } from 'express';
+import express, { Application, Request, Response } from 'express';
+import session from 'express-session';
+import passport from 'passport';
 import cors from 'cors';
 import helmet from 'helmet';
 import swaggerUI from 'swagger-ui-express';
 import logger from '@tablerise/dynamic-logger';
 import autoSwagger from '@tablerise/auto-swagger';
 import SwaggerDocumentDnD5E from '../api-docs/swagger-doc-dungeons&dragons5e.json';
+import SwaggerDocumentUser from '../api-docs/swagger-doc-user.json';
 
 import RoutesWrapper from 'src/routes/RoutesWrapper';
 import DungeonsAndDragonsRouteMiddleware from 'src/routes/middlewares/DungeonsAndDragonsRouteMiddleware';
+import UserRouteMiddleware from 'src/routes/middlewares/UserRouteMiddleware';
 import ErrorMiddleware from 'src/middlewares/ErrorMiddleware';
 
+const COOKIE_AGE = 1000 * 60 * 60 * 120;
```

Comment:
Quão frequente será o uso desse padrão de tempo? Se for constante é interessante colocar numa enum.

Reply:
Vai ser utilizado apenas uma vez no arquivo app para setar o tempo de expiração do token.